src/toolbar/tests/toolbar.test.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import { mount, shallow } from 'enzyme';
+
+import { Toolbar } from '..';
+import { ToolbarButton } from '../ToolbarButton';
+import { ToolbarGroup } from '../Toolbar';
+import { ToolbarKebab } from '../ToolbarKebab';
+import { ToolbarList } from '../ToolbarList';
+import { ToolbarView } from '../ToolbarView';
+
+const toolbarData = require('../data/toolbar-big.json');
+const viewData = require('../data/toolbar-view.json');
+
+describe('Toolbar', () => {
+  it('Well it works ;-)', () => {
+    const t = shallow(<Toolbar
```

Comment:
I would rename `t` to something more meaningful

Reply:
well I like the 't' as toolbar, or the test subject

I think that the short name does not negatively affect readability in this context